src/filter.ts

```diff
@@ -43,7 +43,7 @@ export class Filter {
       return
     }
 
-    const doc = jsyaml.safeLoad(yaml) as FilterYaml
```

Comment:
due to V4.XX release : 
see : https://github.com/nodeca/js-yaml/blob/2cef47bebf60da141b78b085f3dea3b5733dcc12/CHANGELOG.md
> [4.0.0] - 2021-01-03
> - Breaking: removed `safe*` functions. Use `load`, `loadAll`, `dump`
>   instead which are all now safe by default.